ui/src/components/CERN_FOOTER.css

```diff
@@ -22,22 +22,32 @@
   max-width: 1140px;
   width: 100%;
   margin: auto;
+  padding: 55px 3%;
```

Comment:
why do you need these? If we are using ant design should take care of it

Reply:
removed